### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by almarklein**
* Some more examples (multiple cubes, skybox, volume slice)
* Some tweaks along the way.
* The API for the texture view needs some more love ... but probably in another PR.

### CODE REVIEW

**Comment by Korijn** [comment]
Think you forgot to add that new material?

File: examples/volume_slice2.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+"""
+Render slices through a volume, by creating a 3D texture, with 2D views.
+Simple and relatively fast, but no subslices.
```
Comment (Korijn) [inline examples/volume_slice2.py:3]: What are subslices?

File: examples/volume_slice1.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+
+def scroll(degrees):
+    global index
+    index = index + int(degrees / 15)
+    index = max(0, min(nslices - 1, index))
+    im = vol[index]
+    tex.data[:] = im
+    tex.update_range((0, 0, 0), tex.size)
+    material.dirty = 1
```
Comment (Korijn) [inline examples/volume_slice1.py:50]: Why is this needed?

File: examples/volume_slice2.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+canvas = WgpuCanvasWithScroll()
+renderer = vv.renderers.WgpuRenderer(canvas)
+scene = vv.Scene()
+
+vol = imageio.volread("imageio:stent.npz")
+nslices = vol.shape[0]
+index = nslices // 2
+
+tex_size = tuple(reversed(vol.shape))
+tex = vv.Texture(vol, dim=2, size=tex_size, usage="sampled")
```
Comment (Korijn) [inline examples/volume_slice2.py:33]: `dim=2`?

File: examples/volume_slice2.py (hunk 1 above)
Comment (ivoflipse) [reply examples/volume_slice2.py:3]: Interpolation between two slices?

Comment (almarklein) [reply examples/volume_slice2.py:3]: yeah, that's what I meant :)

File: examples/volume_slice1.py (hunk 2 above)
Comment (almarklein) [reply examples/volume_slice1.py:50]: because of how the renderer works right now. That process should be made somewhat smarter.

File: examples/volume_slice2.py (hunk 3 above)
Comment (almarklein) [reply examples/volume_slice2.py:33]: Yes, if I understood this right, the way to do this is to create a 2D texture with size `wxhxn`, with `n > 1`

**Comment by Korijn** [comment]
Awesome! Love the simplicity :)
